DOCKER.md

```diff
@@ -81,15 +72,15 @@ $ docker-compose exec database psql -h database -Upostgres circulate_development
 
 #### Rails console
 ```
-$ docker-compose exec web rails c
+$ docker-compose run web rails c
 ```
 
 ## Testing Suite
 Run the testing suite from within the container:
 
 ```
-$ docker-compose exec web rails test
-$ docker-compose exec web rails test:system
+$ docker-compose run web rails test
+$ docker-compose run web rails test:system
```

Comment:
Changing the system tests command from `exec` (do it in same container) to `run` (do it in a new container) will result in conflicts when trying to run at the same time as the development `web` service.

```
Errno::EADDRINUSE: Address in use - bind(2) for "0.0.0.0" port 4000
```

Related to `exec` vs `run` also, `docker-compose run web` without a `--rm` option will leave behind a bunch of stopped containers with random names.

```
» docker-compose ps -a
             Name                           Command               State                        Ports
-------------------------------------------------------------------------------------------------------------------------
circulate_assets_1               ./docker-entrypoint.sh bin ...   Up       3000/tcp
circulate_database_1             docker-entrypoint.sh postgres    Up       0.0.0.0:5435->5432/tcp
circulate_selenium_chrome_1      /opt/bin/entry_point.sh          Up       4444/tcp, 0.0.0.0:5900->5900/tcp
circulate_web_1                  ./docker-entrypoint.sh bin ...   Up       0.0.0.0:3000->3000/tcp, 0.0.0.0:4000->4000/tcp
circulate_web_run_11d00c616bc5   ./docker-entrypoint.sh rai ...   Exit 1
circulate_web_run_13fa5fa5e61e   ./docker-entrypoint.sh rai ...   Exit 1
circulate_web_run_1c450e235f6d   ./docker-entrypoint.sh rai ...   Exit 0
circulate_web_run_637eeb1cf223   ./docker-entrypoint.sh rai ...   Exit 1
circulate_web_run_a53439855ff1   ./docker-entrypoint.sh rails c   Exit 0
circulate_web_run_e59160a4e8fe   ./docker-entrypoint.sh rai ...   Exit 1
```